vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerIT.java

```diff
@@ -77,6 +78,7 @@ public void selectDateOrTimeSeparately() {
                 message.getText().contains("1987-04-10 14:42"));
 
         picker.setDate(null);
```

Comment:
Should this method also trigger a `change` event on the DateTimePicker?

Reply:
My assumption was that the `change` event for `DateTimePicker` will be triggered automatically with the `change` event triggering on `DatePicker`.

Reply:
Right, but why do we need to call sendKeys then?